website/docs/developer/deployment-guide/deploying-from-windows.mdx

```diff
@@ -70,42 +73,42 @@ This application will be used in future steps to authenticate users.
   Only use the non-lambda subnet id in the data-persistence-tf/terraform.tfvars. In cumulus-tf use both.
   :::
   :::warning
-  Overwrite the ```orca-sandbox``` in ```orca-sandbox-tf-locks``` with your prefix as well.
+  Overwrite the `orca-sandbox` in `orca-sandbox-tf-locks` with your prefix as well.
   :::
 - In rds-cluster-tf/terraform.tfvars
-    - Use values of your choice for ```db_admin_username``` and ```db_admin_password```
-    - Set ```tags``` to ```{ "Deployment" = "PREFIX" }```
-    - Set ```permissions_boundary_arn``` to ```arn:aws:iam::YOUR ACCOUNT ID:policy/NGAPShRoleBoundary```
-    - Add ```rds_user_password = "CumulusD3faultPassw0rd"``` and change as desired.
-    - Set ```provision_user_database``` to ```true```
-    - Set ```cluster_identifier``` to ```"PREFIX-cumulus-db"```
+    - Use values of your choice for `db_admin_username` and `db_admin_password`
+    - Set `tags` to `{ "Deployment" = "PREFIX" }`
+    - Set `permissions_boundary_arn` to `arn:aws:iam::YOUR ACCOUNT ID:policy/NGAPShRoleBoundary`
+    - Add `rds_user_password = "CumulusD3faultPassw0rd"` and change as desired.
+    - Set `provision_user_database` to `true`
+    - Set `cluster_identifier` to `"PREFIX-cumulus-db"`
 
 - In rds-cluster-tf/terraform.tf
-    - Set ```bucket``` to ```"PREFIX-tf-state"```
-    - Set ```key``` to ```"PREFIX/cumulus/terraform.tfstate"```
-    - Set ```dynamodb_table``` to ```"PREFIX-tf-locks"```
+    - Set `bucket` to `"PREFIX-tf-state"`
+    - Set `key` to `"PREFIX/cumulus/terraform.tfstate"`
+    - Set `dynamodb_table` to `"PREFIX-tf-locks"`
 
 - Go to https://github.com/asfadmin/CIRRUS-core/blob/master/Dockerfile and download the file to the same folder as your downloaded repo and orca folder.
   :::tip
   Make sure that no extension is added.
   :::
 - Open a commandline in the same folder.
-    - Run ```docker build -t orca .``` and ```docker run -it --rm -v pathToYourFolder:/CIRRUS-core orca /bin/bash```
+    - Run `docker build -t orca .` and `docker run -it --rm -v pathToYourFolder:/CIRRUS-core orca /bin/bash`
     - The commandline should now be inside a docker container.
-      ```bash
+      `bash
```

Comment:
Since this is a code block, you will likely want three \`\`\` here and at the end of the code block to maintain formatting and language.

Reply:
Gah, knew I'd mistakenly do one of those. Fixed.